docs/testing/testplan.md

```diff
@@ -0,0 +1,427 @@
+We focus on functional test cases. The goal is to report bugs, issues or differences observed from the expected behavior.
+
+## Audience
+
+Any person interested in testing the beta release of the Web Monetization extension.
+
+## Objectives
+
+1. Validating that, once installed, users can configure Web Monetization settings on the extension.
+2. Validating that once the extension is successfully connected to a digital wallet, a user is able to configure payments or disconnect from the wallet.
+3. When the extension facilitates paying a monetized web page, check how payments get distributed between the receiving wallets on the page.
+
+## Dependencies
+
+### Digital Wallets
+
+The main prerequisite for sending or receiving Web Monetization payments is to have a Web Monetization-enabled digital wallet. Web Monetization-enabled digital wallets are provided by licensed service providers. The wallet providers are regulated by the laws of the countries in which they operate.
+
+Below is a list of the available wallet providers if you want to use real money:
+
+- [Fynbos](https://wallet.fynbos.app)
+- [GateHub](https://gatehub.net)
+
+Wallet availability in a specific country, or availability in particular currency depends on the wallet provider.  
+We maintain a list of [Web Monetization compatible wallets](https://webmonetization.org/docs/resources/op-wallets/).
+
+#### Test Digital Wallet
+
+You have the option to use “play” money from the [Interledger Test Wallet application](https://rafiki.money/).  
+Here you can set up an account that is enabled for Web Monetization, and other Interledger functionality, without involving real money.
+
+## Testing
+
+### Environment and Software
+
+The resources and tools that you will need to test Web Monetization depend on your desired role, and what you want to test.
+This test plan only focuses on the perspective of a website visitor who Wants to pay websites that are web monetized.
+
+1. Hardware: Any PC with internet access
+2. Digital wallet: A Web Monetization enabled digital wallet from which payments will be made
+3. Environment & software:  
+   a. Any operating system (OS)
+   b. A Chromium-based Web browser (e.g. Chrome, Edge, Vivaldi, Brave) or Firefox
+
+### Reporting Issues
+
+1. Use any screen recording application to record the steps and results of your testing (e.g. Loom, Camtasia etc.).
+2. File bugs or other issues, and monitor their status and resolution on [GitHub](https://github.com/interledger/web-monetization-extension/issues).
+3. Join the `#webmonetization` channel on [Slack](https://communityinviter.com/apps/interledger/interledger-working-groups-slack).
+
+### Prerequisites
+
+Before you start testing, sign-up for a wallet:
+
+1. The sign-up process depends on your digital wallet provider. Learn more from the [Dependencies](#Dependencies) section.
+2. If you want to use a test wallet, the sign-up process is quick (within minutes).
```

Comment:
Based on the thread yesterday, it seems like the process is no longer quick, because it mimics GateHub's production KYC steps. I believe the main difference is that with test, you don't have to wait for identity verification (and it's not real money 😁)